### DISCUSSION TITLE
Display inline <svg> with NextJS as the src of an <Image> element

### DISCUSSION THREAD

**Discussion by bhaktatejas922** [vercel/next.js#41290] [Help]
I've tried everything I could find online. Anyone know how to do this? 

I tried  @svgr/webpack, and more. I'm not importing the svg, because it is made on the fly. 

Here's an example svg 
`<svg xmlns="http://www.w3.org/2000/svg" width="300" height="170" viewBox="0.15994477272033691,0.38479520975618237,0.00004029273986816406,0.00005054473876947574"> <polyline className="shape" points="0.15994477272033691,0.3848362177517878 0.15994596481323248,0.3848343104031551 0.15998506546020508,0.38484575449495184 0.1599581241607666,0.38479520975618237 0.15994477272033691,0.3848362177517878"></polyline> </svg>`

I tried doing this 

```
const GeojsonSVG = () =>  {
<svg xmlns="http://www.w3.org/2000/svg" width="300" height="170" viewBox="0.15994477272033691,0.38479520975618237,0.00004029273986816406,0.00005054473876947574"> <polyline className="shape" points="0.15994477272033691,0.3848362177517878 0.15994596481323248,0.3848343104031551 0.15998506546020508,0.38484575449495184 0.1599581241607666,0.38479520975618237 0.15994477272033691,0.3848362177517878"></polyline> </svg> }

<Image
  key={index}
  src={<GeojsonSVG/>}
  alt={title}
  layout="fill"
  objectFit="cover"
/>
}
```

as well as 

```
const GeojsonSVG = () =>  {
return (
<svg xmlns="http://www.w3.org/2000/svg" width="300" height="170" viewBox="0.15994477272033691,0.38479520975618237,0.00004029273986816406,0.00005054473876947574"> <polyline className="shape" points="0.15994477272033691,0.3848362177517878 0.15994596481323248,0.3848343104031551 0.15998506546020508,0.38484575449495184 0.1599581241607666,0.38479520975618237 0.15994477272033691,0.3848362177517878"></polyline> </svg> )}

<Image
  key={index}
  src={GeojsonSVG()}
  alt={title}
  layout="fill"
  objectFit="cover"
/>
}
```

neither works

**Comment by bhaktatejas922** [comment]
I get this 

`
Image is missing required "src" property: <img src=​"data:​image/​gif;​base64,R0lGO…5BAEAAAAALAAAAAABAAEAAAIBRAA7" decoding=​"async" data-nimg=​"fill" style=​"position:​ absolute;​ inset:​ 0px;​ box-sizing:​ border-box;​ padding:​ 0px;​ border:​ none;​ margin:​ auto;​ display:​ block;​ width:​ 0px;​ height:​ 0px;​ min-width:​ 100%;​ max-width:​ 100%;​ min-height:​ 100%;​ max-height:​ 100%;​ object-fit:​ cover;​">​
`

**Comment by artrixdotdev** [comment]
bump

**Comment by bhaktatejas922** [comment]
Doing this works 
```
  let svg = '<svg xmlns="http://www.w3.org/2000/svg" width="300" height="170" viewBox="0.15994477272033691,0.38479520975618237,0.00004029273986816406,0.00005054473876947574"> <polyline className="shape" points="0.15994477272033691,0.3848362177517878 0.15994596481323248,0.3848343104031551 0.15998506546020508,0.38484575449495184 0.1599581241607666,0.38479520975618237 0.15994477272033691,0.3848362177517878"></polyline> </svg>';
  let blob = new Blob([svg], {type: 'image/svg+xml'});
  let url = URL.createObjectURL(blob);
  
  ```
  
  and inputing the url to the src of the Image component. However this is hack I'd prefer not to do
